src/test/features/response/draft/responseDraft.ts

```diff
@@ -47,33 +47,53 @@ describe('ResponseDraft', () => {
       expect(responseDraftModel.moreTimeNeeded).to.be.eql(undefined)
       expect(responseDraftModel.isMoreTimeRequested()).to.be.eq(false)
     })
-    describe('requireDefence', () => {
-      it('should return false when no response type set', () => {
-        const responseDraftModel: ResponseDraft = new ResponseDraft()
+  })
 
-        expect(responseDraftModel.requireDefence()).to.be.eq(false)
-      })
-      it('should return false when response type is OWE_NONE and counter claim is true', () => {
-        const responseDraftModel: ResponseDraft = new ResponseDraft()
-        responseDraftModel.response = new Response(ResponseType.OWE_NONE)
-        responseDraftModel.counterClaim = new CounterClaim(true)
+  describe('requireDefence', () => {
+    it('should return false when no response type set', () => {
```

Comment:
Either this method is named poorly or there is a bug in its logic. I would expect a method named `requireDefence` to return true if the response is not set.